lib/register.js

```diff
@@ -1,26 +1,4 @@
 'use strict';
```

Comment:
I don't completely remember, but I think there was something with backwards compatibility which was why we left it as it is, can be wrong though